bendigobank.rb

```diff
@@ -2,69 +2,79 @@
+   connect
+   fetch_accounts
+   fetch_transactions
+    puts @accounts
+    close
   end
 
+  binding.pry
+  browser = Watir::Browser.new(:chrome)
+  browser.goto('https://demo.bendigobank.com.au')
+
   def login
-    # here you log in to the bank
-    browser = Watir::Browser.new(:chrome, 'goog:chromeOptions' => { detach: true })
-    browser.goto('https://demo.bendigobank.com.au')
-    # sign in script
-    browser.button(:tabindex => "4", :class => "input_submit",
-      :name => "customer_type",:value =>"personal").click
+    browser.button(:tabindex => "4", :class => "input_submit", :name => "customer_type",:value =>"personal").click
   end
 
   def fetch_accounts
-  # fetch html data using nokogiri, take only fragment of html.
-  html = Nokogiri::HTML.fragment(browser.div(class: 'accounts-list').html)
+    html = Nokogiri::HTML.fragment(browser.div(class: "content__wrapper").html)
   parse_accounts(html)
   end
 
-  def fetch_transactions
-  @accounts.each do |account|
-    two_months_ago = (Date.today - 60).strftime('%d/%m/%Y')
-    today = Date.today.strftime('%d/%m/%Y')
-    StatementPage.new(@browser)
-                 .fetch_transactions_into(@accounts,
-                                          from_date: two_months_ago,
-                                          to_date: today)
-    parse_transactions(account, html)
-    end
+  def parse_transactions(account, html)
+    transactions = []
+    date = html.at_css("[data-semantic-group] h3").text
+    description = 'transaction'
+    #description = html.at_css('h2.panel__header__label__primary[data-semantic="transaction-title"] span')
+    amount = html.at_css('span.amount.debit[data-semantic="amount"]').text
+    currency = 'USD'
+    account_name = html.at_css("h6.grouped-list__group__heading").text
+    transactions = ['date'=> date, 'description'=> description, 'amount'=> amount,'currency' => currency, 'account_name'=> account_name]
+    transactions = Transactions.new( date,  description, amount,  currency, account_name )
+    puts transactions.to_json
   end
 
   def parse_accounts(html)
     # Iterate accounts unsing css selectors
-    html.css('ol.grouped-list__group__items li').each do |li|
-      name = html.at_css("h6[data-semantic='account-group-heading']").text
-      balance = html.at_css("span[data-semantic='available-balance']").text
-      currency =html.at_css("span[data-semantic='available-balance'][0]").text
-      nature = html.at_css("div._3jAwGcZ7sr _5KR4Am_fPD").text
-
-      account = Account.new(name, balance, currency, nature ) # create account here
+    accounts = []
+    html.css('ol.grouped-list grouped-list--compact grouped-list--indent li').each do |html|
+      name = html.at_css("h6.grouped-list__group__heading").text
+      balance = html.at_css('[data-semantic="available-balance"]').text
+      currency = 'USD'
+      nature = 'credit card'
+      transactions = Transactions.new( date,  description, amount,  currency, account_name )
```

Comment:
You have no information about transactions in `parse_accounts`. 
Currently it is blank array. `[]`